doc/newsboat.asciidoc

```diff
@@ -393,6 +393,20 @@ other readers like FeedHQ. Thus, one can leverage Newsboat's support for FeedHQ
 See <<_passwords_for_external_apis>> for other, more secure ways to store your
 password.
 
+=== Feedbin
+
+_Supported since Newsboat TODO_.
```

Comment:
I need to add a version number here

Reply:
2.35 will come out at the end of March, I'm sure this code will be long merged by then.